### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by leqiao-1**
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`

### CODE REVIEW

File: olive/passes/pytorch/qat_utils.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 # --------------------------------------------------------------------------
 import copy
 
+import pytorch_lightning
 import torch
 import torch.quantization.quantization_mappings as tqqm
+from packaging import version
```
Comment (trajepl) [inline olive/passes/pytorch/qat_utils.py:10]: Sometime, there will be bug for packaing.version.  Suggest to use `from distutils.version import LooseVersion`

`if LooseVersion(torch.__version__) >= LooseVersion("1.12.0"):`

Comment (jambayk) [reply olive/passes/pytorch/qat_utils.py]: image
python gives this warning to me

Comment (leqiao-1) [reply olive/passes/pytorch/qat_utils.py]: Before we met this issue with packaging.version
image
I will have a check.

Comment (trajepl) [reply olive/passes/pytorch/qat_utils.py]: In Python 3.10 and 3.11, distutils will be formally marked as deprecated. Currently it is 3.7/3.8 for olive?

Comment (jambayk) [reply olive/passes/pytorch/qat_utils.py]: I got the warning above in python 3.8 (3.8.15).

Comment (jambayk) [reply olive/passes/pytorch/qat_utils.py]: Looking at the docs, the example version you shared above doesn’t follow the PEP standards. So do we expect to see versions like those in practice? 
The version above looks like one created for a custom build of torch?

Comment (jambayk) [reply olive/passes/pytorch/qat_utils.py]: Actually, it looks like the version comparison result above is expected? https://packaging.pypa.io/en/stable/version.html 

The first version is a prerelease (1.11.0a) so it is earlier than the release version (1.11.0).

if we want to get it true for prerelease versions also, we could use ‘version.parse(“1.11.0a0+git”).release’ >= ‘version.parse(“1.11.0”).release’?

Comment (leqiao-1) [reply olive/passes/pytorch/qat_utils.py]: I think we can keep version.parse as before.

Comment (jambayk) [reply olive/passes/pytorch/qat_utils.py]: Yes, I agree. The issue you mentioned is expected behavior so I don’t think it’s really an issue.

Comment (leqiao-1) [reply olive/passes/pytorch/qat_utils.py:10]: last commit reverted

**Review by jambayk — APPROVED** [review]

**Pull request merged**